jbpm-task-assigning/jbpm-task-assigning-core/src/test/java/org/jbpm/task/assigning/model/solver/realtime/RemoveTaskProblemFactChangeTest.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.task.assigning.model.solver.realtime;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+import org.jbpm.task.assigning.model.Task;
+import org.jbpm.task.assigning.model.TaskAssigningSolution;
+import org.junit.Test;
+
+import static org.junit.Assert.assertFalse;
+
+public class RemoveTaskProblemFactChangeTest extends BaseProblemFactChangeTest {
```

Comment:
The same concerns as for the other tests in the `realtime` package.

Reply:
more or less same explanation if you refer to the random tests